packages/compass-components/src/components/form-field-container.tsx

```diff
@@ -1,15 +1,16 @@
-import { spacing, css, cx } from '@mongodb-js/compass-components';
+import { spacing } from '@leafygreen-ui/tokens';
+import { css, cx } from '@leafygreen-ui/emotion';
 import React from 'react';
 
 const formFieldContainerStyles = css({
-  marginTop: spacing[4],
-  width: spacing[7] * 6,
+  margin: `${spacing[4]}px 0`,
 });
 
 function FormFieldContainer({
   className = '',
   children,
 }: {
+  key?: string;
```

Comment:
`key` is an implicit react component prop, you shouldn't need to specify it here explicitly (and it wouldn't really be passed down to component when you do something like `<Component key="1" />` anyway)

Reply:
I learned that while working on this and just forgot to clean it up. Will fix.